kpm/tasks/package.rake

```diff
@@ -37,6 +37,10 @@ namespace :package do
 
   desc 'Install gems to local directory'
   task :bundle_install do
+    # cleanup
```

Comment:
Should we maybe instead have the `:bundle_install` task depend on `:clean`, which does the same?

Reply:
The only way I could reproduce the error was packaging a couple of version without cleaning the vendor folder. Could not reproduce the error otherwise. Adding the cleaning of that folder resolves that issue. 

I will update the way it cleans to depend on the task :clean.

I will add the version into the Gemfile, then a check to verify if the version exits in the gem repository before creating the package. If not exists, show a message stating that the version is not found.